docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -69,6 +71,14 @@ $ kubectl get dataimportcron cron-test -o yaml | grep -B 5 storageClassName
         storageClassName: rook-ceph-block
 ```
 
+When no storage class is defined, DataVolume controller will not create PVCs, and we'll get the event:
+`DataVolume.storage spec is missing accessMode and no storageClass to choose profile`  
+Moreover, currently even after defining a default storage class, DataVolume will not get reconciled so it will remain stuck in this state.
+
+The way to resolve it manually is to delete the stuck DataVolume(s) after setting the default storage class. The DataImportCron controller will then immediately re-create the DataVolume using the default storage class.
+
+NOTE: Once this [PR](https://github.com/kubevirt/containerized-data-importer/pull/2196) gets in, when default storage class is set, DataVolume will be automatically recovered.
+
```

Comment:
We need to be thinking about installed versions that have and do not have the fix.  Instead of the above I would say something like:

Once a default storage class has been defined, updated versions of CDI should resolve this issue automatically.  If it does not resolve within x minutes, manually delete the affected DataVolumes.  CDI will then recreate them with the proper reference to the default storage class.

Might want to provide a command that shows how the affected DVs can be found.

Reply:
Much better, I'll update.

Reply:
/lgtm